mongodb-advanced/components/people-component.js

```diff
@@ -6,25 +6,33 @@ export default {
   templateUrl: 'mongodb-advanced/people-component.html'
 };
 
-function Ctrl($scope, exPeopleService) {
+function Ctrl($scope, exPeopleService, brAlertService) {
   const self = this;
   self.showMyModal = false;
 
-  exPeopleService.getPeople(people => {
-    self.savedPeople = people;
-  });
+  self.getPeople = function() {
+    exPeopleService.getPeople(people => self.savedPeople = people)
```

Comment:
either `return exPeopleService...` so that a promise is returned here, or remove the enclosing curly braces which has the same effect.